genie-docs/src/docs/asciidoc/_metrics.adoc

```diff
@@ -410,6 +410,30 @@ and system metrics and statistics.
 |DiskCleanupTask
 |-
 
+|genie.health.endpoint.timer
+|Time taken for the Health endpoint to collect and aggregate state from health indicators
+|nanoseconds
+|HealthCheckMetricsAspect
+|status, exceptionClass
+
+|genie.health.indicator.timer
+|Time taken for each health indicator to report its status
+|nanoseconds
+|HealthCheckMetricsAspect
+|status, exceptionClass, healthIndicatorClass
+
+|genie.health.indicator.counter
+|Counter for of calls each health indicator tagged by reported status
+|count
+|HealthCheckMetricsAspect
+|status, exceptionClass, healthIndicatorName
```

Comment:
Just minor that you're using `healthIndicatorName` as a tag in these latter two and `healthIndicatorClass` in the previous one. Is there an explicit reason this is different?

Reply:
I wish we could be consistent, but in one case (the timer) all I see is the class.
In the other (the counter) i get a map with name->status.